app/src/utils/eventId.ts

```diff
@@ -0,0 +1,61 @@
+export default class EventId {
+    public static eventIdToZoneId(eventId: number): number {
+        switch (eventId) {
+            case 1:
+                return 2;
+            case 2:
+                return 6;
+            case 3:
+                return 4;
+            case 4:
+                return 8;
+            case 7:
+                return 4;
+            case 8:
+                return 4;
+            case 9:
+                return 4;
+            case 10:
+                return 2;
+            case 11:
+                return 2;
+            case 12:
+                return 2;
+            case 13:
+                return 6;
+            case 14:
+                return 6;
+            case 15:
+                return 8;
+            case 16:
+                return 8;
+            case 17:
+                return 8;
+            case 31:
+                return 2;
+            case 32:
+                return 6;
+            case 33:
+                return 4;
+            case 34:
+                return 8;
+            case 123:
+            case 124:
+            case 125:
+                return 2;
+            case 126:
+            case 127:
+            case 128:
+                return 6;
+            case 129:
+            case 130:
+            case 131:
+                return 8;
+            case 132:
+            case 133:
+            case 134:
+                return 4;
+        }
+        return -1;
```

Comment:
To make this more readable, should probably utilize a constant of some sort to make the numbers more verbose for the zones, e.g. `return Zones::INDAR`

Reply:
Small idea, these files contain data available from the api. It might a good idea to setup some code generator that is able to generate the body of the function.

Also for the default case I would throw an Exception.

Reply:
Yes actually that's a very good point. 

http://census.daybreakgames.com/s:CENSUSID/get/ps2/metagame_event?c:limit=200 should be all you need

Reply:
Ahhh I did not find this API while looking through the data.
Thx for the tip

Reply:
What's your preferred package for http requests?

Reply:
@marci4 If this was PHP I'd say Guzzle, but that's PHP only unfortunately. Will deferr this to @microwavekonijn

Reply:
Axios was what I always used. However it might be better to use node-fetch as it is a lot simpler(and the census api doesn't use fancy http codes overall). There was another one that I saw that looked very promising however I have to get back to you on that.

Note: I am going to add all the census api endpoints to the ps2census package with appropriate functionality w.r.t. when the service is unavailable or when it is dead.

Reply:
#27